server/lib/plans.ts

```diff
@@ -95,6 +95,19 @@ export async function handleHostPlanBankTransfersLimit(host, { throwException =
   }
 }
 
+export async function handleTransferWisePaymentLimit(host, expense, { throwException = false }): Promise<void> {
+  const hostPlan = await host.getPlan();
+  if (
+    throwException &&
+    hostPlan.payedWithTransferWiseLimit !== null &&
+    hostPlan.payedWithTransferWise + expense.amount >= hostPlan.payedWithTransferWiseLimit
```

Comment:
We stopped doing that. You're now allowed to go over the limit, but you can't anymore once you're over limit. So you can simply remove the `+ expense.amount`

Reply:
Nice, I'll fix it and rebase, thanks!